deploy/helm/airflow-operator/crds/crds.yaml

```diff
@@ -455,18 +455,27 @@ spec:
                       items:
                         properties:
                           authenticationClass:
-                            description: Name of the [AuthenticationClass](https://docs.stackable.tech/home/nightly/concepts/authentication) used to authenticate users.
+                            description: |-
+                              Name of the [AuthenticationClass](https://docs.stackable.tech/home/nightly/concepts/authentication) used to authenticate users.
+
+                              To get the concrete [`AuthenticationClass`], we must resolve it. This resolution can be achieved by using [`ClientAuthenticationDetails::resolve_class`].
```

Comment:
For me personally this is a regression (as Rustdoc should not leak into the CRD).
But this needs to be changed in op-rs, not blocking this PR on that (feel free to resolve this thread)

Reply:
https://github.com/stackabletech/operator-rs/pull/1030 will adress it